### ISSUE TITLE
PodLogs CRDs are being disabled.  

### ISSUE THREAD

**Issue by bentonam** [grafana/k8s-monitoring-helm#567]
The comment [values.yaml#L2059](https://github.com/grafana/k8s-monitoring-helm/blob/main/charts/k8s-monitoring/values.yaml#L2059)

```
# Skip installation of the Grafana Alloy CRDs, since we don't use them in this chart
```

Is incorrect, the crds chart deploys the PodLogs object https://github.com/grafana/alloy/tree/main/operations/helm/charts/alloy/charts/crds which is supported by this chart and may need to be enabled in some situations if the PodLogs CRD is being used.

**Comment by petewall** [comment]
Yeah, the comment could be improved. Maybe we should say something along the lines of:

"Alloy CRDs are disabled by default, but can be enabled to deploy the definitions for PodLogs objects"

I don't want to enable them by default because we've run into too much trouble with the operator CRDs and upgrade conflicts, deployments with things like ArgoCD, etc...
